internal/exec/stages/files/filesystemEntries.go

```diff
@@ -132,6 +132,15 @@ func (s *stage) createCrypttabEntries(config types.Config) error {
 	return nil
 }
 
+type Report struct {
```

Comment:
Let's not export this struct for now, since there's nothing outside this package that uses it.